tests/model_hub_tests/torch_tests/test_timm.py

```diff
@@ -75,7 +75,31 @@ def teardown_method(self):
 
     @pytest.mark.parametrize("name", ["mobilevitv2_050.cvnets_in1k",
                                       "poolformerv2_s12.sail_in1k",
-                                      "vit_base_patch8_224.augreg_in21k"])
+                                      "vit_base_patch8_224.augreg_in21k",
+                                      # TODO: will revert
```

Comment:
Please leave one of these models in the list after revert, so we will have it in precommit for future

Reply:
Done.